lib/game.rb

```diff
@@ -77,9 +36,9 @@ def game_flow
     p2_mark = "O"
     p1_type, p2_type = player_types[game_mode]
     winning_move_sequences = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]]
-    until game_is_over?(winning_move_sequences, p1_winning_move_sequence) || game_is_over?(winning_move_sequences, p2_winning_move_sequence) || @grid.get_available_moves(current_grid) == []
+    until game_over?(winning_move_sequences, p1_winning_move_sequence) || game_over?(winning_move_sequences, p2_winning_move_sequence) || @grid.get_available_moves(current_grid) == []
```

Comment:
It's because each player has different move_sequences but yes, you are right. This was one of the things I wanted remove. Once I remove move_sequences then I'll be able to use the grid throughout.